cocos/asset/assets/texture-2d.ts

```diff
@@ -94,6 +96,36 @@ export class Texture2D extends SimpleTexture {
         return this._mipmaps;
     }
     set mipmaps (value) {
+        if (value[0].mipmapLevelDataSize && value[0].mipmapLevelDataSize.length > 0) {
```

Comment:
How about add type information for parameter `value`? Is it an `not empty Array` absolutely?

Reply:
sure